chutney/action-impl/src/test/resources/logback-test.xml

```diff
@@ -19,10 +19,14 @@
     <logger name="liquibase" level="WARN"/>
     <logger name="LiquibaseSchemaResolver" level="WARN"/>
     <logger name="org.apache" level="WARN"/>
```

Comment:
to remove

Reply:
I don't know what you want here ?
I'm aiming here to reduce tests' logs.